File: Python/1012.py
values = input().split(" ")

a, b, c = values
a, b, c = float(a), float(b), float(c)

print("TRIANGULO: {:.3f}".format(a * c / 2))
print("CIRCULO: {:.3f}".format(3.14159 * c**2))
print("TRAPEZIO: {:.3f}".format((a + b) * c / 2))
print("QUADRADO: {:.3f}".format(b**2))
print("RETANGULO: {:.3f}".format(a * b))

File: JavaScript/1012.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split(" ");

a = parseFloat(lines[0]);
b = parseFloat(lines[1]);
c = parseFloat(lines[2]);

console.log("TRIANGULO: " + ((a * c) / 2).toFixed(3));
console.log("CIRCULO: " + (3.14159 * (c * c)).toFixed(3));
console.log("TRAPEZIO: " + (((a + b) * c) / 2).toFixed(3));
console.log("QUADRADO: " + (b * b).toFixed(3));
console.log("RETANGULO: " + (a * b).toFixed(3));
